okta/duo/spec/package.json

```diff
@@ -5,7 +5,9 @@
   "license": "MIT",
   "scripts": {
     "ci:test": "jest",
-    "lint": "tslint -c ../../../tslint.json src/*.ts src/**/*.ts"
+    "lint": "tslint -c ../../../tslint.json src/*.ts src/**/*.ts",
+    "clean": "echo",
```

Comment:
required to add these empty scripts. There is an open issue with yarn v1 - https://github.com/yarnpkg/yarn/issues/7121